### ISSUE TITLE
body feature extractor

### ISSUE THREAD

**Issue by Andrew-Brown1** [movienet/movienet-tools#8]
Hi,

Thanks for making all the code available. I have a question about the person feature extractor. Is it the one that gets the best results in the paper? e.g. Table C4 (body | r50-softmax | 32.81)

**Comment by ycxioooong** [comment]
Yes, it's the best checkpoint pre-trained on MovieNet itself.

**Comment by Andrew-Brown1** [comment]
great, thank you
